cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -494,7 +494,7 @@ Searching for Metadata
 CDAP supports searching metadata of entities. To find which applications, datasets, streams, etc. have a particular
 metadata property or metadata tag, submit an HTTP GET request::
 
-  GET <base-url>/namespaces/<namespace>/metadata/search?query=<term>&target=<entity-type>
+  GET <base-url>/namespaces/<namespace>/metadata/search?query=<term>&target=<entity-type>[&target=<entity-type2>...]
```

Comment:
Good question. @poornachandra : is the first ``target`` optional, and does it default to `all` if it is not there?